src/components/ProductsPageSection/index.js

```diff
@@ -29,7 +29,7 @@ export default function ProductPageSection({
   const desktopWideViewPort = isBrowser && 1215 < width && width <= 1407;
   const desktopUltraWideViewPort = isBrowser && 1408 < width;
 
-  const userDocsLink = `/documentation/${title.toLowerCase()}`;
+  const userDocsLink = `https://www.overture.bio/documentation/${title.toLowerCase()}`;
```

Comment:
adding `www` isn't necessary anymore, opting for a simpler url pattern makes it more memorable.
```suggestion
  const userDocsLink = `https://overture.bio/documentation/${title.toLowerCase()}`;
```